apps/opik-backend/src/main/java/com/comet/opik/domain/ImageUtils.java

```diff
@@ -4,24 +4,33 @@
 
 public class ImageUtils {
     public static final String PREFIX_JPEG = "/9j/";
-    public static final String PREFIX_PNG = "iVBORw0KGgo=";
-    public static final String PREFIX_GIF0 = "R0lGODlh";
-    public static final String PREFIX_GIF1 = "R0lGODdh";
+    public static final String PREFIX_PNG = "iVBORw0KGgo";
+    public static final String PREFIX_GIF0 = "R0lGODlh"; // new version gif (89a)
+    public static final String PREFIX_GIF1 = "R0lGODdh"; // old version gif (87a)
     public static final String PREFIX_BMP = "Qk";
-    public static final String PREFIX_TIFF0 = "SUkqAA";
-    public static final String PREFIX_TIFF1 = "II*";
+    public static final String PREFIX_TIFF0 = "SUkq"; // little endian tiff
+    public static final String PREFIX_TIFF1 = "TU0A"; // big endian tiff
     public static final String PREFIX_WEBP = "UklGR";
 
-    private static final String IMAGE_CHARS = "[^\"]+";
-    private static final String IMAGE_TRUNCATION_REGEX = "data:image/[^;]{3,4};base64," + IMAGE_CHARS + "|"
-            + PREFIX_JPEG + IMAGE_CHARS + "|"
-            + PREFIX_PNG + IMAGE_CHARS + "|"
-            + PREFIX_GIF0 + IMAGE_CHARS + "|"
-            + PREFIX_GIF1 + IMAGE_CHARS + "|"
-            + PREFIX_BMP + IMAGE_CHARS + "|"
-            + PREFIX_TIFF0 + IMAGE_CHARS + "|"
-            + PREFIX_TIFF1 + IMAGE_CHARS + "|"
-            + PREFIX_WEBP + IMAGE_CHARS;
+    private static final String IMAGE_CHARS = "[A-Za-z0-9+/]+={0,2}";
+    private static final String IMAGE_TRUNCATION_REGEX =
```

Comment:
I think we can assume that the images are going to be always as a separate string field in the JSON, meaning that you could assume that it'll start with `"` before the whole possible options (`/"(data:image|iVBORw0KGgo|...).*"/`)

Reply:
Correct but then the `"` will be replaced as part of the "regex replace" function which will break the JSON structure

Reply:
Correct, then you can replace by `"[image]"` instead of just `[image]`

Reply:
Done in https://github.com/comet-ml/opik/pull/615/commits/1e4afb4456f01df1d2cb4a6cb8e565584d10acfa